autogen.sh

```diff
@@ -1,75 +0,0 @@
-#!/bin/sh
-
-set -e
-
-oldpwd=$(pwd)
-
-if [ -n "$MESON_DIST_ROOT" ]; then
-    topdir="$MESON_DIST_ROOT"
-else
-    topdir=$(dirname $0)
-fi
-
-cd $topdir
-
-gtkdocize --docdir libkmod/docs || touch libkmod/docs/gtk-doc.make
```

Comment:
This does not work with release tarball if gtk-doc is not installed. `libkmod/docs/gtk-doc.make` is already a symbolic link:
```
libkmod/docs/gtk-doc.make -> /usr/share/gtk-doc/data/gtk-doc.flat.make
```

Same is true for m4/gtk-doc.m4:
```
m4/gtk-doc.m4 -> /usr/share/aclocal/gtk-doc.m4
```

Reply:
We would need an `rm -f` call for both mentioned files before touch to make sure that it always works.

Reply:
and there's no point fixing it in git since it's only reproducible when building from the tarball, correct?

Reply:
Correct, that's why I didn't notice it during tests.

Reply:
See https://github.com/kmod-project/kmod/pull/295